src/shared/helpers/getCourseDate.ts

```diff
@@ -0,0 +1,6 @@
+export const getCourseDate = (startDate: string) => {
+  const fourteenDays = 14 * 24 * 60 * 60 * 1000;
+  const isStale = Date.parse(startDate) < Date.now() - fourteenDays;
+
+  return isStale ? 'TBD' : startDate;
+};
```

Comment:
We alredy have `dayjs` package. So, please, use it for all kind of date manipulations

https://day.js.org/